app/src/services/market_maker.ts

```diff
@@ -143,6 +146,42 @@ class MarketMakerService {
     }
   }
 
+  static getActualPriceWithHoldings = (holdings: BigNumber[]): number[] => {
+    return calcPrice(holdings)
+  }
+
+  getBalanceInformationWithMultipleOutcomes = async (
+    ownerAddress: string,
+    outcomeQuantity: number,
+  ): Promise<BigNumber[]> => {
+    const conditionId = await this.getConditionId()
+    const collateralTokenAddress = await this.getCollateralToken()
+
+    const balances = []
+    logger.log(`Outcomes quantity ${outcomeQuantity}`)
+    for (let i = 0; i < outcomeQuantity; i++) {
+      const collectionId = await this.conditionalTokens.getCollectionIdForOutcome(
+        conditionId,
+        1 << i,
+      )
+      logger.log(
```

Comment:
These logs are too detailed, I don't think they should be at the `log` level. Maybe `debug` is more appropriate.

Reply:
Done!